hack/rp-dev/rp_funcs.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash -e
+######## Helper file with the functions to fully deploy RP dev either locally or using Azure DevOps Pipelines ########
+
+source hack/devtools/rp_dev_helper.sh
+
+setup_rp_config() {
+  # Check if exactly non-empty two arguments are provided
+  if [[ $# -ne 2 ]]; then
+    echo "Error: $0 <AZURE_PREFIX> <SKIP_DEPLOYMENTS>. Please try again"
+    exit 1
+  fi
+
+  # TODO - check if needed
+  export USER=dummy
+  # Export the SECRET_SA_ACCOUNT_NAME environment variable and run make secrets
+  az account set -s fe16a035-e540-4ab7-80d9-373fa9a3d6ae
+  export SECRET_SA_ACCOUNT_NAME=rharosecretsdev && make secrets
+
+  source hack/devtools/rp_dev_helper.sh && verify_downloading_secrets
+
+  export LOCATION=eastus
+  # Source environment variables from the secrets file
+  source secrets/env
+  echo -e "Success step 1 ✅ - Directory '$expected_dir' has been created with files - ${secret_files[@]}\n"
+
+  export AZURE_PREFIX=$1 SKIP_DEPLOYMENTS=$2 NO_CACHE=true ARO_INSTALL_VIA_HIVE=true ARO_ADOPT_BY_HIVE=true DATABASE_NAME=ARO
```

Comment:
1. Convert these to lowercase, they are used internally right?
2. Pass them into your functions as arguements, that way your functions aren't relying on the external environment and side affects (you can pass them to `az` instead of having `az` rely on the environment)
3. Use double quotes

Reply:
1. I don't think so, since these vars were exported. See the resulting _env-int_ file that we export at step 4 from https://github.com/Azure/ARO-RP/blob/master/docs/deploy-full-rp-service-in-dev.md#deploying-an-int-like-development-rp
2. >(you can pass them to az instead of having az rely on the environment) 

Can you elaborate on that?
3. Done